foundry/game/File.py

```diff
@@ -1,3 +1,4 @@
+from ctypes import Structure, sizeof, addressof, memmove, c_char, c_ubyte
```

Comment:
The advantage of ctypes is, that we don't have to manually parse out the header?

Reply:
Yes, the advantage of `ctypes` is that you can define your structure and then basically just cast the data to it rather than using something like `struct` to `pack`/`unpack`/etc. In my opinion, it looks a lot nicer to show the structure definition rather than the format string you get via `struct` or having to interpret the manual setting of `self.whatever` if unpacking manually.